src/Streamly/Coreutils/Uniq.hs

```diff
@@ -12,11 +12,11 @@ module Streamly.Coreutils.Uniq
 import Control.Monad.Catch (MonadThrow)
 import Data.Char (isSpace, toLower)
 import Streamly.Data.Stream (Stream)
+import Streamly.Internal.Data.Stream (catRights)
```

Comment:
Why Internal? You can just remove this import and use Stream.catRights, it is available in the qualified import of Stream.